Trident/offsetfinder.c

```diff
@@ -16,6 +16,7 @@
 t_target_environment target_environment = NotSupported;
 
 t_target_environment info_to_target_environment(const char *device_model, const char *system_version) {
+    determineTarget("iPhone5,4", "9.3.2", iPhone52_iOS921);
```

Comment:
Can you insert your device after iPhone53_iOS933 everywhere please? We try to keep offsetfinder ordered.